README.md

```diff
@@ -69,7 +69,7 @@ _Note: The `publicPath` property is required, whereas all other options are opti
 ### methods
 
 Type: `Array`  
-Default: `[ 'GET' ]`
+Default: `undefined`
```

Comment:
I think this change seems to be breaking change. Is this wrong?

Reply:
@hiroppy I don't think so here is the check : 
```
const acceptedMethods = context.options.methods;

    if (acceptedMethods && acceptedMethods.indexOf(req.method) === -1) {
      return goNext();
    }
```
and in tests I changed the `post` request test that supposed to `return 404` in the previous settings to be like `get` now it passes the test without issues. 

I changed the documentation a little in the last commit, so now it makes more sense.

Reply:
Yes, it is not breaking change, it is feature